src/Text/Regex/Base/Context.hs

```diff
@@ -385,7 +385,7 @@ instance (RegexLike a b) => RegexContext a b (AllTextMatches (Array Int) (MatchT
                  (mts) -> return . AllTextMatches . listArray (0,pred $ length mts) $ mts
 
 instance (RegexLike a b) => RegexContext a b (AllTextMatches [] b) where
-  match r s = AllTextMatches [ fst (ma!0) | ma <- matchAllText r s ]
+  match r s = AllTextMatches [ fst (ma ! 0) | ma <- matchAllText r s ]
```

Comment:
Let's add spaces to all occurences of `ma!0` (there are five more).

Reply:
Ok. (A few of them are in comments...)